docs/statsig-warehouse-native/features/autotune.md

```diff
@@ -2,6 +2,7 @@
 title: Autotune (Beta)
 slug: /statsig-warehouse-native/features/autotune
 sidebar_label: Autotune
+displayed_sidebar: cloud
```

Comment:
already has a sidebar

https://docs.statsig.com/statsig-warehouse-native/features/autotune

Reply:
removed it!